lib/ecto/migration.ex

```diff
@@ -245,7 +245,9 @@ defmodule Ecto.Migration do
   end
 
   @doc """
-  Adds a column when creating or altering a table.
```

Comment:
Remember to keep the first line of the documentation short as it is used as a summary by ExDoc. :)